src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -294,6 +294,9 @@ void Resizer::estimateParasitics(ParasiticsSrc src,
       global_router_->estimateRC(spef_writer.get());
       parasitics_src_ = ParasiticsSrc::global_routing;
       break;
+    case ParasiticsSrc::detailed_routing:
+      parasitics_src_ = ParasiticsSrc::detailed_routing;
+      break;
```

Comment:
Is it right that this does no estimation?

Reply:
The way I'm implementing it on ORFS, we already load the SPEF file after writing it with RCX, so no estimation is needed. Perhaps in the future we want to call rcx and load the spef data only in C++, and here would be a correct place for it.

Maybe I can add a comment about these future plans here?